### ISSUE TITLE
Any option to increase the maximum perceived roughness of the new PMREM generator?

### ISSUE THREAD

**Issue by Domenicobrz** [mrdoob/three.js#18504]
I'm using the following code along with the new PMREM generator:
https://pastebin.com/pvkXtazH

To simply render a mesh with a single diffuse texture, by setting the roughness value to 1 

The result is:

![Annotazione 2020-01-29 034231](https://user-images.githubusercontent.com/25647854/73324230-9733f800-424a-11ea-9690-52d03e3428fd.jpg)

I'm not familiar with the inner workings of PMREM but the computed irradiance felt "shiny" so I tried to increase the sampling theta for the PMREM blur shader by multiplying it by 1.5:

`float theta = dTheta * float(dir * i) * 1.5;`

Now the end result is a lot similiar to the output I get on eevee on Blender (using the standard shader with only the roughness slider set to the maximum value):

![test2](https://user-images.githubusercontent.com/25647854/73324498-7324e680-424b-11ea-85dc-92015b1792c3.jpg)

The question is: am I doing anything wrong in the code i shared on pastebin that would cause the perceived irradiance to look "shiny" ?

If not, is there any option I can pass to PMREM to increase the blurriness of the prefiltered map without having to change the base library?

**Comment by WestLangley** [comment]
@elalish will have to answer, but to speed things up, it _may_ by helpful if you provide the dimensions of your equirectangular texture.

**Comment by Domenicobrz** [comment]
@WestLangley 1024 x 512

[env.zip](https://github.com/mrdoob/three.js/files/4126027/env.zip)

**Comment by Domenicobrz** [comment]
I'll add more comparisons with different hdr maps to hopefully convey a bit better the problem I'm having

All tests are done with the same MeshPhysicalMaterial, metalness is always set to 0 and roughness is always set to the maximum value 1

The pictures on the left used the standard result from PMREM, the pictures on the right had the blurring theta angle multiplied by 1.5 as explained in my original post


![comp1](https://user-images.githubusercontent.com/25647854/73370275-40fb9f00-42b4-11ea-8af8-33fd87c2383a.jpg)
![comp2](https://user-images.githubusercontent.com/25647854/73370285-435df900-42b4-11ea-9cd8-42b62babc112.jpg)


The particular material I'm trying to recreate benefits from the increased "maximum perceived roughness", but I can't find a way to get this result without modifying the base library, is there something that could help my use case that I'm missing?

**Comment by elalish** [comment]
@Domenicobrz Thank you for the detailed comparisons! First, PMREM is very much approximate, and the implementations vary widely in the types of approximations used. Many use spherical harmonics for rough objects, which makes them appear *more* matte than they really should be. I'm not sure if that's what Eevee uses, but keep in mind that neither of these is ground truth. 

To make up for these approximations, I've often seen environment maps chosen with brighter lights than is realistic to help give rough objects some contrast. This shouldn't be necessary with this PMREM, so you might try lowering the dynamic range of the environment a bit, or blurring it.

**Comment by mrdoob** [comment]
Would it be better to compare with Arnold?

**Comment by looeee** [comment]
I'm not sure how easy it is to set up environment maps in Arnold but I'll give it a go. 

@Domenicobrz can you share the model?

**Comment by Domenicobrz** [comment]
@looeee Here's the .glb mesh, diffuse map and the hdr envmaps I'm using

[test.zip](https://github.com/mrdoob/three.js/files/4135682/test.zip)

I'd like to make clear that I'm **not** saying the computed irradiance of the current PMREM is incorrect, the purpose of this issue was to find a way to recreate the look of the particular material I'm trying to simulate without having to modify the base library, since it seems that particular use-case benefits from the increased theta angle in the blur shader used by pmrem

**Comment by WestLangley** [comment]
@Domenicobrz I expect you need a different material to properly solve your problem. Currently, you are using a material designed specifically to model plastics and metals.

For more info, see https://google.github.io/filament/Filament.html#materialsystem/clothmodel.

**Comment by Mugen87** [comment]
Related: https://threejs.org/examples/webgl_materials_physical_sheen

**Comment by Domenicobrz** [comment]
@WestLangley thank you a lot! that's definitely going to help :) I'll look into the sheen BRDF
feel free to close this task, that should answer my question

**Comment by looeee** [comment]
> @looeee Here's the .glb mesh, diffuse map and the hdr envmaps I'm using

Thanks, I can't load .glb files into 3DS Max though. Can you export from Blender as binary FBX? 
Seems Blender.org is down at the moment otherwise I'd install Blender and do that myself. 

I realize that this issue is kind of resolved now but it would still be useful to compare our output to Arnold.

**Comment by mrdoob** [comment]
> Seems Blender.org is down at the moment otherwise I'd install Blender and do that myself.

https://twitter.com/blender_org/status/1223045762478723072

**Comment by looeee** [comment]
Here's the result of a render from Arnold. Using a standard arnold shader, all default settings except roughness = 1, metalness = 0. 

I was unable to re-export the file from Blender, so I used Paint3D and exported an OBJ file. Normals did not make it through this process which might be why the folds look rather different in this render. 

As @sciecode pointed out, it's kind of hard to compare these renders since there is different tonemapping being used. In three.js it's probably ACESFilmic, no idea in Blende, and in Max tonemapping is called "Exposure control":

![exp](https://user-images.githubusercontent.com/5307958/73524321-7254a600-443f-11ea-8a87-04518dcc51cd.png)

I set this render to "no exposure control" and saved as a PNG with default gamma. 

![out](https://user-images.githubusercontent.com/5307958/73524404-a4fe9e80-443f-11ea-8b54-7898e78269bc.png)
